pkg/standalone/uninstall.go

```diff
@@ -11,8 +11,22 @@ func Uninstall() error {
 		"docker", "rm",
 		"--force",
 		DaprPlacementContainerName)
+
+	errorMessage := ""
+	if err != nil {
+		errorMessage += "Could not delete Dapr Placement Container - it may not have been running "
+	}
+
+	err = utils.RunCmdAndWait(
```

Comment:
Someone would like to keep their data in redis... I think we need to provide option. as we talked yesterday,

e.g. only if they give --all option, cli will delete all containers including redis.
```
dapr uninstall --all 

```

how do you think about this suggestion? 

cc/ @yaron2

Reply:
Implemented now.